src/qt/bitcoin.qrc

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE RCC><RCC version="1.0">
+<RCC>
```

Comment:
is this change necessary

Reply:
My concern with this kind of Qt Creator generated diff junk is that it forces Gui developers to
manually edit files to eliminate this stuff. This - I suspect - has contributed to bringing Gui dev to a crawl.
Technically this is valid Qt Creator generated code. I would petition members to reevaluate the policy concerning 
Qt Creator diff junk.

--- 

Please consider my comments here:

In this situation Qt Creator generates elements that it is expecting:
When I removed them it logged this error.

https://github.com/bitcoin-core/gui/pull/135#discussion_r535574885

---

PS I am beginning to suspect that when elements are missing that are expected by Qt - it triggers some kind of error handling in the Qt framework.
This is probably wasting cycles and is slowing down the over all performance of the Gui itself.
Only a theory at this point - I haven't been had time to research it further.